WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -33,32 +34,32 @@ regtest.rpcport = 18443
 regtest.rpcuser = 7c9b6473600fbc9be1120ae79f1622f42c32e5c78d
 regtest.rpcpassword = 309bc9961d01f388aed28b630ae834379296a8c8e3
 ```
-4. Start Bitcoin core with: bitcoin-qt.exe -regtest
-5. Do not worry about "Syncing Headers" just press the Hide button.
-6. Go to MainMenu / Window / Console
-7. Generate a new address with:
+5. Start Bitcoin Core with: bitcoin-qt.exe -regtest.
+6. Do not worry about "Syncing Headers" just press the Hide button.
```

Comment:
```suggestion
6. Do not worry about "Syncing Headers" just press the Hide button. Because you run on Regtest, no Mainnet blocks will be downloaded.
```